### PR TITLE
Add scroll view

### PR DESCRIPTION
**Pull request by kevromster**
* Show scrollbars on main views when content doesn't fit.
* Automatic scrolling down on Chat and Activity views, also in the case when a user enters new messages.
* Enable scrollbars in Completion popup.
* Hide scrollbars if the content fits into the window, also in Linux Chrome.

### CODE REVIEW

File: packages/sparkling-controls/src/ScrollView.svelte [hunk 1]
```diff
@@ -11,12 +11,28 @@
...
+
+	import { afterUpdate } from 'svelte'
+
 	export let stylez: string = ''
 	export let scrollPosition: number = 0
+	export let autoscroll: boolean = false
+
+	let container: HTMLElement
+
+	afterUpdate(() => {
```
Comment (haiodo) [inline packages/sparkling-controls/src/ScrollView.svelte:23]: Why not use $: { ... }

**Comment by haiodo** [comment]
@kevromster should we also customise scrolls for windows/linux?

File: packages/sparkling-controls/src/ScrollView.svelte (hunk 1 above)
Comment (kevromster) [reply packages/sparkling-controls/src/ScrollView.svelte:23]: Approach with afterUpdate() was in Svelte tutorial, I'll try with $: (seems to be more correct)

**Comment by kevromster** [comment]
On Linux scrollbars look good in Firefox and a little ugly in Chrome, but we see them only on big content. Don't have Windows to try right now. I think we could improve visibility later.

File: packages/sparkling-controls/src/ScrollView.svelte (hunk 1 above)
Comment (kevromster) [reply packages/sparkling-controls/src/ScrollView.svelte:29]: Changed to `$:` to correctly update by 'scrollPosition' in CompletionPopup. Autoscrolling still needs to be run in afterUpdate().

**Pull request merged**